railgun/internal/diagnostics/server.go

```diff
@@ -46,8 +63,28 @@ func (s *Server) Listen(ctx context.Context) error {
 	}
 }
 
-// installHandlers adds the Profiling webservice to the given mux.
-func installHandlers(mux *http.ServeMux) {
+// receiveConfig watches the config update channel
+func (s *Server) receiveConfig(ctx context.Context) {
+	for {
+		select {
+		case dump := <-s.ConfigDumps.Configs:
+			if dump.Failed {
+				successfulConfigDump = dump.Config
+			} else {
+				failedConfigDump = dump.Config
```

Comment:
```suggestion
				successfulConfigDump = dump.Config
```

Reply:
Flipped the condition instead in 8e7c9866